src/sql/parts/notebook/models/cell.ts

```diff
@@ -230,13 +230,15 @@ export class CellModel implements ICellModel {
 				}
 			}
 		} catch (error) {
+			let message: string;
 			if (error.message === 'Canceled') {
 				// swallow the error
+				message = localize('executionCanceled', 'Query execution is canceled');
```

Comment:
How about 'Query execution was canceled'? Is canceled is a little weird grammar-wise.